app/models/host/base.rb

```diff
@@ -234,6 +233,16 @@ def set_interfaces(parser)
       self.interfaces.reload
     end
 
+    def set_reported_data(parser)
+      return unless parser.boot_time
+
+      reported_data_facet.update!(boot_time: parser.boot_time)
```

Comment:
I would guess you need to do 
```suggestion
      reported_data_facet.update!(boot_time: Time.at(parser.boot_time))
```

Reply:
yes, this helped, I prefer to store datetime in the DB for the searching, with integer that would be harder...